packages/rlc-common/src/metadata/buildTsConfig.ts

```diff
@@ -13,7 +13,8 @@ const restLevelTsConfigInAzureSdkForJs: (
       compilerOptions: {
         module: "NodeNext",
         moduleResolution: "NodeNext",
-        rootDir: "."
+        rootDir: ".",
+        skipLibCheck: true
```

Comment:
introduced by https://github.com/Azure/azure-sdk-for-js/issues/30962#issuecomment-2321925251